app/src/main/kotlin/com/wire/android/ui/home/conversations/search/SearchPeopleViewModel.kt

```diff
@@ -261,15 +261,31 @@ abstract class SearchPeopleViewModel(
         }
     }
 
+    /**
+     * Do not under any circumstances call this function in a loop.
+     * It will cause the to emit a value each time. This in turn
+     * will result in synchronous execution of functions on the main
+     * thread like to use outdated values.
+     */
```

Comment:
Then we should also rename it accordingly for redundancy to something like `addSingleContactToGroup(contact: Contact)` ?

Reply:
It is single. ContacT